hack/e2e.go

```diff
@@ -146,21 +146,21 @@ func main() {
 		}
 	}
 
-	failure := false
+	var success bool
```

Comment:
If neither `*ctlCmd` or `*test` are set then this will exit non-zero... no? If I simply want to `-up` or `-down` a cluster it should be able to report whether it was successful or not.